db/seeds.rb

```diff
@@ -1,37 +1,31 @@
+require 'factory_bot_rails'
+include FactoryBot::Syntax::Methods
+
 puts "Seeding started..."
 
-user = User.create(
-  name: 'nikola',
-  email: 'user@test.com',
-  password: 'password'
-)
-
-30.times do
-
-  giveaway =  user.giveaways.build(
-    title: Faker::Commerce.product_name,
-    description: Faker::Lorem.paragraphs,
-    location: Faker::Address.city,
-  )
-  giveaway.save
-
-  img = %w[lamp.jpg seat.jpg].sample
-  pic = Picture.create(
-    title: img[0..-5],
-    giveaway_id: giveaway.id
-  )
-
-  pic.image.attach(io: File.open(Rails.root.join("app/assets/images/#{img}")), filename: "#{img}")
-
-  rand(2..5).times do |c|
-    giveaway.comments.build(
-        body: Faker::Movies::Lebowski.quote,
-        user_id: rand(1..User.count)
+rand(2..5).times do |i|
+  user = create(:user)
+
+
+  rand(4..7).times do |n|
+    giveaway = create(:giveaway, user: user)
+
+      img = %w[lamp.jpg seat.jpg].sample
+      pic = Picture.create(
+        title: img[0..-5],
+        giveaway_id: giveaway.id
       )
 
-    giveaway.save
-  end
+      pic.image.attach(io: File.open(Rails.root.join("app/assets/images/#{img}")), filename: "#{img}")
 
+      rand(2..5).times do |c|
+        giveaway.comments.build(
```

Comment:
I believe you can use` create!` directly, instead of building the record, and then saving it.

Reply:
@violeta-p Yes, I was getting error with a previous refactor that required giveaways to be build explicitly, but now it is safe to use create. Thank you